### DISCUSSION TITLE
Firefox excluded

### DISCUSSION THREAD

**Discussion by mtengmo** [Romanitho/Winget-AutoUpdate#103] [Q&A]
Why is Firefox excluded by default ? 
Is it a risk to enable it? I have a lot of machines still on old versions.

**Comment by Romanitho** [comment]
The Excluded list is given as an example. You can adjust it.
By default I remove the browsers because they have a self update engine that manage updates better than just reinstall a newer version

**Comment by mtengmo** [comment]
Thanks, I seen issue with updating these also, error from the installation because of admin/user context. 
Discord.Discord
Spotify.Spotify

**Reply by Romanitho** [reply]
so you probably need to add them to your excluded list
